mitodl/mit-open
# resource category

### What are the relevant tickets?
closes https://github.com/mitodl/hq/issues/4727

### Description (What does it do?)
This pr adds resource_category as a serializer field and a filter for the /learning_resource and /learning_resources_search apis. It removes is_learning_material stays from the 


### How can this be tested?
Run docker-compose run web ./manage.py recreate_index --all

Go to http://localhost:8063/api/v1/learning_resources/
Verify that the resource_category filter works, for example http://localhost:8063/api/v1/learning_resources/?resource_category=learning+material

Got to http://localhost:8063/api/v1/learning_resources_search
Verify that the resource_category filter works, for example http://localhost:8063/api/v1/learning_resources_search/?resource_category=learning+material
Verify that aggregations work with resource_category http://localhost:8063/api/v1/learning_resources_search/?aggregations=resource_category

Go to http://localhost:8063/api/v1/schema/redoc and verify that the documentation looks good
